bindings/nodejs/lib/types/block/payload/transaction/essence.ts

```diff
@@ -57,10 +60,7 @@ class RegularTransactionEssence extends TransactionEssence {
     })
     outputs: Output[];
 
-    @Type(() => Payload, {
-        discriminator: PayloadDiscriminator,
-    })
-    payload: Payload | undefined;
+    payload?: unknown;
```

Comment:
I know nothing about TS but this looks sus :(

Reply:
`unknown` is a reserved keyword in TS, from [w3schools](https://www.w3schools.com/typescript/typescript_special_types.php#:~:text=unknown%20is%20best%20used%20when,is%20of%20the%20casted%20type.) 

> unknown is best used when you don't know the type of data being typed. To add a type later, you'll need to cast it.
Casting is when we use the "as" keyword to say property or variable is of the casted type.

And this is what we want here because we don't know what type `payload` will be.  Also using `Payload` isn't really of any use because it's an abstract class that shouldn't be constructed but used internally to cast to the subclasses.